pipelines/h6c/idr2/v3/lstbin/lstbin-template.toml

```diff
@@ -26,6 +26,11 @@ timeout = "48h"
 [LSTBIN_OPTS]                                                                                                                                                                                                                                                                                                                                               
 # ---- Parameters used by hera_opm to build the makeflow ----
 outdir = "${LSTAVG_OPTS:outdir}"
+{% if REDAVG -%}
+bl_chunk_size = 5000
+{% else %}
+bl_chunk_size = 0
+{% endif %}
```

Comment:
I wasn't aware you could do conditionals in tomls... can you explain in a comment how this works? Also, where is `REDAVG` defined, I don't immediately see it.

Reply:
AH yeah, this is a Jinja2 template that gets filled out by a CLI command. This is explained in the comment at the top of the TOML. I've added a bit more information to it.